src/main/java/seedu/java/util/Duke.java

```diff
@@ -0,0 +1,50 @@
+package seedu.java.util;
+
+import java.util.Scanner;
+
+/**
+ * The main class. Contains Storage, TaskList & Ui class. Relies on Parser class to handle user's input.
+ * run()
+ * main()
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+    public static String CLOSE = "0";
+
+    /**
+     * A Duke constructor that loads up a file based on a file path.
+     * @param filePath - a path to take data from, .txt format
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (Exception e) {
```

Comment:
Pokemon catching!! I think it might be better to specify the exact exceptions you want to catch (e.g. FileNotFoundException) and also for those methods where you just throw Exception, might be better to be more specific.